File: app/src/main/java/acr/browser/lightning/constant/Proxy.java
package acr.browser.lightning.constant;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * Proxy choice integer definition.
 * <p>
 * These should match the order of @array/proxy_choices_array
 */
@IntDef({Constants.NO_PROXY, Constants.PROXY_ORBOT, Constants.PROXY_I2P, Constants.PROXY_MANUAL})
@Retention(RetentionPolicy.SOURCE)
public @interface Proxy {}

File: app/src/main/java/acr/browser/lightning/constant/Proxy.kt
package acr.browser.lightning.constant

import androidx.annotation.IntDef


/**
 * Proxy choice integer definition.
 *
 * These should match the order of @array/proxy_choices_array
 */
@IntDef(NO_PROXY, PROXY_ORBOT, PROXY_I2P, PROXY_MANUAL)
@Retention(AnnotationRetention.SOURCE)
annotation class Proxy
